spanner/test/opentelemetry/test/ot_metrics_test.go

```diff
@@ -259,10 +259,12 @@ func TestOTMetrics_GFELatency(t *testing.T) {
 	t.Cleanup(func() {
 		te.Unregister(ctx)
 	})
-	spanner.EnableOpenTelemetryMetrics()
 	server, client, teardown := setupMockedTestServerWithConfig(t, spanner.ClientConfig{OpenTelemetryMeterProvider: te.mp})
 	defer teardown()
 
+	// enabling OpenTelemetry metrics after spanner client initialization
+	spanner.EnableOpenTelemetryMetrics()
```

Comment:
this will panic in spanner client version <= v1.60.0

Reply:
Hmmm... I'm wondering whether this is partly 'works as intended'. Not the panic, of course, but:
1. `spanner.EnableOpenTelemetryMetrics()` sets a global variable that enables it for __all__ clients.
2. Calling `spanner.EnableOpenTelemetryMetrics()` after having created one or more clients will then enable it for all those clients, right? What will become the behavior of those clients after calling `spanner.EnableOpenTelemetryMetrics()`? Also, what happens to any client that was created and using OpenCensus before this method gets called?

Would it maybe make more sense to do the following:
1. Set a global variable the first time that a client is created.
2. Return an error if someone tries to call `spanner.EnableOpenTelemetrymetrics()` after a client has been created.
3. Also return an error if someone tries to create a client with an `OpenTelemetryMetricProvider` or any other OpenTelemetry-related configuration without first having called `spanner.EnableOpenTelemetryMetrics()`.

The reason that I think that the above is a more reasonable approach is that I don't think it is clear to users what happens with existing clients if you call `spanner.EnableOpenTelemetryMetrics()` at a random point in time in your application.